### PR TITLE
Lt 1525 mod clearance permission

### PR DESCRIPTION
**Pull request by stephanieboyd**

### CODE REVIEW

File: applications/tests/tests_finalise_application.py [hunk 1]
```diff
@@ -35,12 +35,48 @@ def test_gov_user_finalise_permission_error(self):
...
+
+        self.gov_user.role = self.role
+        self.gov_user.role.permissions.set(
+            [GovPermissions.MANAGE_CLEARANCE_FINAL_ADVICE.name, GovPermissions.MANAGE_LICENCE_DURATION.name]
+        )
+        self.gov_user.save()
+
+        data = {"licence_duration": 13}
+        url = reverse("applications:finalise", kwargs={"pk": clearance_application.pk})
+        response = self.client.put(url, data=data, **self.gov_headers)
```
Comment (andytaylorkainos) [inline applications/tests/tests_finalise_application.py]: no response body to assert?

File: applications/views/applications.py [hunk 2]
```diff
@@ -242,6 +242,7 @@ class ApplicationManageStatus(APIView):
     @transaction.atomic
     def put(self, request, pk):
         application = get_application(pk)
+        is_licence_application = application.application_type != "exhibition_clearance"
```
Comment (andytaylorkainos) [inline applications/views/applications.py]: is there not an enum for this?

File: applications/libraries/licence.py [hunk 3]
```diff
@@ -8,6 +8,8 @@ def get_default_duration(application):
 
     Rules defined in: https://uktrade.atlassian.net/browse/LT-1586
     """
+    if application.application_type == ApplicationType.EXHIBITION_CLEARANCE:
+        return DefaultDuration.TEMPORARY.value
 
     if application.export_type == ApplicationExportType.TEMPORARY:
```
Comment (Mark-Spiers-kainos) [inline applications/libraries/licence.py]: update to elif please

File: applications/views/applications.py (hunk 2 above)
Comment (stephanieboyd) [reply applications/views/applications.py]: Ah yes, thank you :)

File: applications/libraries/licence.py (hunk 3 above)
Comment (stephanieboyd) [reply applications/libraries/licence.py]: Changed

File: applications/tests/tests_finalise_application.py (hunk 1 above)
Comment (stephanieboyd) [reply applications/tests/tests_finalise_application.py]: Added an assertion for the licence_duration. Good spot

**Pull request merged**